components/templates/Pattern.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Alert from '../atoms/Alert';
+import MDContent from "./MDContent";
+import styles from '../../styles/PatternLayout.module.css';
+
+export default function Pattern(props) {
+
+    return (
+        <MDContent markdown={props.pattern.markdown} pattern={props.pattern} t={props.t}>
+            <Alert>
+                <p>{props.t["This pattern is currently"]} <strong>{props.pattern.data.status}</strong></p>
+            </Alert>
+            <img className={styles.featuredImage} src={props.pattern.data["feature-img-path"]} alt={props.pattern.data["feature-img-alt"]}/>
+            <p><strong>{props.pattern.data["short-description"]}</strong></p>
+        </MDContent>
```

Comment:
Same as above, templates should not be relying on other templates

Reply:
This is part of the implementation we agreed on in our conversation, I thought that was clear.
Ultimately, the code functions quite well in this setup - it reduces duplication while improving separation of concerns. That is the core goal of this PR.

Reply:
I just opened a new PR to resolve this based off my understanding from our conversation